### PR TITLE
Ft/teams

### PR DESCRIPTION
**Pull request by AdelineA**
- Add bg image
- add heading
- Add text
- Add video Image
- add heading
- add a list of skills
- add heading
- Add both mobile and desktop responsiveness

### CODE REVIEW

File: build/sections/team.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
+<div class="container text-white pb-12">
+   <div class="lg:w-[80%] w-full pt-20 sm:pt-28 lg:pt-32">
+      <h1 class="font-raleway text-xl md:text-2xl lg:text-[36px] font-[700]">Unsere <span class="text-primary-green">Teams</span>  </h1>
```
Comment (gasangw) [inline build/sections/team.html]: - [OPTIONAL]  Kindly always check the Tailwindcss document for some features you are unsure of exist. For example, you declared the font-weight of `700` like `font-[700]` yet TailwindCSS has an inbuilt class for that which is `font-bold`. For more information about font weight feel free to check out this link. [font-weight](https://tailwindcss.com/docs/font-weight).

File: build/sections/team.html [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      <p class=" font-montserrat text-base sm:text-lg lg:text-[20px] font-medium pt-6">Wir stellen für Sie Teams je nach Ihrem Bedarf in einer Größe von 3-7 Entwickler:innen zusammen. Alle Entwickler:innern besitzen bei uns das gleiche Profil an Softwareentwicklungsskills und können Software in einem agilen Projektumfeld für Kunden mit europäischen Standards in Bezug auf Anforderungen und Qualität entwickeln.</p>
+   </div>
+   <div class="pb-10">
+         <p class="font-raleway pb-6 font-medium text-base sm:text-lg lg:text-[20px] pt-10">Einblick in die M<span class="border-b-4 border-primary-green shadow-xl">otivation und den Drive u</span>nserer Talente:</p>
```
Comment (gasangw) [inline build/sections/team.html]: - [OPTIONAL]  Kindly always check the Tailwindcss document for some features you are unsure of that exist. For example, you declared the font size of `20px` like `text-[20px]` yet TailwindCSS has an inbuilt class for that which is `text-sm`. For more information about font size feel free to check out this link. [font-size](https://tailwindcss.com/docs/font-size)

File: build/sections/team.html [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+            <li class="bg-white px-2 py-1.5 rounded-full">Nest.JS</li>
+            <li class="bg-white px-2 py-1.5 rounded-full">ORM</li>
+            <li class="bg-white px-2 py-1.5 rounded-full">Prisma</li>
+            <li class="bg-white px-2 py-1.5 rounded-full">GraphQL</li>
+            <li class="bg-white px-2 py-1.5 rounded-full">CI/CD</li>
+            <li class="bg-white px-2 py-1.5 rounded-full">Scrum Framework</li>
+         </ul>
+      </div>
+      <div>
+         <h3 class="text-primary-green text-[16px] font-[900] pt-5">ZUM TECH-SKIlLPROFIL</h3>
```
Comment (gasangw) [inline build/sections/team.html]: - Kindly check out the comments above on inbuilt classes of `font size` and `font weight`. They can help you improve this line of code well.

**Review by gasangw — CHANGES_REQUESTED** [review]
Hi @AdelineA 

# CHANGES REQUIRED 🟢
 
this-close-taylor-strecker

- Great work done on the desktop version ✅ 
- Great design of the medium screen and mobile design. ✅ 

## However below are the changes required 🔴 

- This is your design. Kindly compare it with the figma design below this. I have highlighted a few areas to change and marked them out for you.

Screenshot (21)

- Figma design is here. Compare it with your above design and make a few changes

Screenshot (22)

- Kindly re-structure this section of the skill set to look well on the mobile section. There are many gaps between the skillset buttons.

Screenshot (25)

**Review by UmutesiMMA — CHANGES_REQUESTED** [review]
image
image


I suggest arranging those list items in  such a way that the circled space is not  big
and increasing the bottom padding of you div (after the ZUM TECH ....) for better visibility in mobile view.
image


otherwise, it looks great!!🤝🏾🤝🏾
leonardo-di-caprio-wolf-of-wall-street

**Review by gasangw — APPROVED** [review]
Hi @AdelineA 

## Congratulations 💯 

excellent-work-inside-job

- Thanks for Implementing the changes I had requested. To me its now complete and ready for merging.

**Review by UmutesiMMA — APPROVED** [review]

**Pull request merged**